_sass/elements/_hex-icon.scss

```diff
@@ -0,0 +1,43 @@
+:root {
+  --sin-60: 0.866;
+}
+
+.hex-icon {
+  --icon-height: 48px;
+
+  width: calc(var(--icon-height) * var(--sin-60));
+  height: var(--icon-height);
+  display: inline-block;
+  margin-right: 12px;
```

Comment:
**nitpick:** This margin is odd. It doesn't seem to be part of the icon itself but depend on the context it's placed in.
Removing that aligns with the above suggestion to combine icon and description into an intermediary container. This whitespace should be applied to that inner container.